src/InputGroup/InputGroup.js

```diff
@@ -1,272 +1,79 @@
-                                    {glyph ? (
-                                        <Icon
-                                            disableStyles={disableStyles}
-                                            glyph={glyph}
-                                            role='presentation' />
-                                    ) : (
-                                        addon
-                                    )}
-                                </span>
-                            )}
-                            <FormInput
-                                {...inputProps}
-                                className={inputClasses}
-                                compact={compact}
-                                disableStyles={disableStyles}
-                                id={inputId}
-                                name={inputName}
-                                onChange={this.handleTextChange}
-                                value={this.state.value} />
-                        </div>
-                    );
-                } else {
-                    return (
-                        <div
-                            {...props}
-                            className={inputGroupClasses}>
-                            <FormInput
-                                {...inputProps}
-                                className={inputClasses}
-                                compact={compact}
-                                disableStyles={disableStyles}
-                                id={inputId}
-                                name={inputName}
-                                onChange={this.handleTextChange}
-                                value={this.state.value} />
-                            {actions ? (
-                                <span className={addonClasses}>
-                                    {children}
-                                </span>
-                            ) : (
-                                <span className={addonClasses}>
-                                    {glyph ? (
-                                        <Icon
-                                            disableStyles={disableStyles}
-                                            glyph={glyph}
-                                            role='presentation' />
-                                    ) : (
-                                        addon
-                                    )}
-                                </span>
-                            )}
-                        </div>
-                    );
-                }
-            }
-        }
+        return (
+            <div
+                {...props}
+                className={inputGroupClasses}
+                compact={compact}>
+                {children}
+            </div>
+        );
     }
 }
 
+InputGroup.Addon = InputGroupAddon;
+InputGroup.Input = FormInput;
+
 InputGroup.displayName = 'InputGroup';
 
 InputGroup.propTypes = {
     actions: PropTypes.bool,
```

Comment:
@SAP/fundamental-react-team Would anyone be in favor of renaming this while we have a breaking change on our hands?
IMO this would be clearer as `hasActions`; right now it sounds like it should take an array of actions.

Reply:
OR do we need `actions` at all? Is this always just related to "is this thing a button?" 😛

Reply:
🔥

Reply:
no actions for the winnnnnn 🏅

Reply:
🔥